src/main/java/mong/Mong.java

```diff
@@ -0,0 +1,358 @@
+package mong;
+
+import mong.exception.IllegalTaskFormatException;
+import mong.exception.IllegalTaskTypeException;
+import mong.task.Deadline;
+import mong.task.Event;
+import mong.task.Task;
+import mong.task.TaskType;
+import mong.task.Todo;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+
```

Comment:
Have multiple classes for better abstraction. This is handling storage and task operations as well.